adds a simple widget that displays posts based from post category

carl-alberto opened the pull request.

cralberto11 committed:
adds a simple widget that displays posts based from post category

cralberto11 committed:
Merge branch 'master' into feature_cpt-widgets

cralberto11 committed:
Merge branch 'master' into feature_cpt-widgets

cralberto11 committed:
- Eliminated Travis CI errors
- Updated code to adhere with WP standards

Kenshino reviewed (changes requested):
Hey, your widget is not applying the before and after widget html tags, so the styles don't work and it's also throwing notices.

See http://wphelphub.com/, I will leave it as it is for a few hours so you can inspect it and correct your code

cralberto11 committed:
Merge branch 'master' into feature_cpt-widgets

cralberto11 committed:
updated translation file

cralberto11 committed:
added WPCS: XSS OK
before_widget
after_widget
before_title
after_title
Not sure on how to properly escape html tags
